src/qml/components/StorageLocations.qml

```diff
@@ -22,11 +22,21 @@ ColumnLayout {
         ButtonGroup.group: group
         text: qsTr("Default")
         description: qsTr("Your application directory.")
-        customDir: optionsModel.getDefaultDataDirString
-        checked: optionsModel.dataDir === optionsModel.getDefaultDataDirString
+        checked: true
```

Comment:
`qrc:/qml/components/StorageLocations.qml:19:5: QML OptionButton: Binding loop detected for property "customDir"`

select custom datadir -> select default -> select custom datadir -> default keeps being selected in GUI

Reply:
done! Used new function to mitigate "default reset" issues

Reply:
> default keeps being selected in GUI

this is still an issue 
(6d4c111c4028525a4071cf03f6d73c8b2e579b8f)

Reply:
Thanks for checking this... The issue is now addressed in [7a34b0d](https://github.com/bitcoin-core/gui-qml/commit/7a34b0da39c0fe63b56981787a8177f492febd1a)